app/experimenter/experiments/api/v5/queries.py

```diff
@@ -28,7 +28,18 @@ def resolve_experiments(root, info):
 
     def resolve_experiment_by_slug(root, info, slug):
         try:
-            return NimbusExperiment.objects.get(slug=slug)
+            return (
+                NimbusExperiment.objects.select_related("reference_branch", "owner")
+                .prefetch_related(
+                    "documentation_links",
+                    "feature_configs",
+                    "countries",
+                    "locales",
+                    "languages",
+                    "changes",
+                )
```

Comment:
I see, sneaking in the detail page changes in with the landing page fixes eh? ;D

If you wanna do both the landing and detail page in one ticket, just update the PR/commit title/message to say it's performance improvements for both.

Also we have the [NimbusExperimentManager](https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/models.py#L34) to store commonly used queries, and since we'll probably want to reuse these prefetch queries in multiple places, it's probably a good idea to stick them over there.  Maybe this could be a new method for getting an individual prefetched experiment?

*Actually*, just looking at the debug for the detail page, it looks like a bunch of things are still getting hit multiple times, so maybe we should pull this into a separate PR that digs into those in more details so we can get the landing page change landed now since that'll be a huge quality of life improvement for everybody:

```SQL
app_1             | DEBUG 2022-06-30 17:53:24,498 django.db.backends (0.002) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."username" = 'dev@example.com' LIMIT 21; args=('dev@example.com',)
nginx_1           | 172.18.0.1 - - [30/Jun/2022:17:53:24 +0000] "GET /nimbus/progressive-247-task-force HTTP/1.1" 200 821 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:103.0) Gecko/20100101 Firefox/103.0" "-"
app_1             | INFO 2022-06-30 17:53:24,509 request.summary 
kinto_1           | Thu Jun 30 17:53:24 2022 - *** uWSGI listen queue of socket "127.0.0.1:45273" (fd: 3) full !!! (35194416/64) ***
app_1             | DEBUG 2022-06-30 17:53:24,628 django.db.backends (0.001) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."username" = 'dev@example.com' LIMIT 21; args=('dev@example.com',)
app_1             | INFO 2022-06-30 17:53:24,631 request.summary 
app_1             | WARNING 2022-06-30 17:53:24,631 django.request Not Found: /sockjs-node
nginx_1           | 172.18.0.1 - - [30/Jun/2022:17:53:24 +0000] "GET /sockjs-node HTTP/1.1" 404 3553 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:103.0) Gecko/20100101 Firefox/103.0" "-"
app_1             | DEBUG 2022-06-30 17:53:24,675 django.db.backends (0.001) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."username" = 'dev@example.com' LIMIT 21; args=('dev@example.com',)
app_1             | DEBUG 2022-06-30 17:53:24,686 django.db.backends (0.002) SELECT "experiments_nimbusfeatureconfig"."id", "experiments_nimbusfeatureconfig"."name", "experiments_nimbusfeatureconfig"."slug", "experiments_nimbusfeatureconfig"."description", "experiments_nimbusfeatureconfig"."application", "experiments_nimbusfeatureconfig"."owner_email", "experiments_nimbusfeatureconfig"."schema", "experiments_nimbusfeatureconfig"."read_only" FROM "experiments_nimbusfeatureconfig" ORDER BY "experiments_nimbusfeatureconfig"."name" ASC; args=()
app_1             | DEBUG 2022-06-30 17:53:24,700 django.db.backends (0.002) SELECT DISTINCT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" INNER JOIN "experiments_nimbusexperiment" ON ("auth_user"."id" = "experiments_nimbusexperiment"."owner_id") WHERE "experiments_nimbusexperiment"."id" IS NOT NULL ORDER BY "auth_user"."email" ASC; args=()
app_1             | DEBUG 2022-06-30 17:53:24,706 django.db.backends (0.001) SELECT "base_locale"."id", "base_locale"."code", "base_locale"."name" FROM "base_locale" ORDER BY "base_locale"."name" ASC; args=()
app_1             | DEBUG 2022-06-30 17:53:24,714 django.db.backends (0.001) SELECT "base_country"."id", "base_country"."code", "base_country"."name" FROM "base_country" ORDER BY "base_country"."name" ASC; args=()
app_1             | DEBUG 2022-06-30 17:53:24,728 django.db.backends (0.001) SELECT "base_language"."id", "base_language"."code", "base_language"."name" FROM "base_language" ORDER BY "base_language"."name" ASC; args=()
app_1             | INFO 2022-06-30 17:53:24,816 request.summary 
nginx_1           | 172.18.0.1 - - [30/Jun/2022:17:53:24 +0000] "POST /api/v5/graphql HTTP/1.1" 200 108616 "https://localhost/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:103.0) Gecko/20100101 Firefox/103.0" "-"
app_1             | DEBUG 2022-06-30 17:53:24,910 django.db.backends (0.001) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."username" = 'dev@example.com' LIMIT 21; args=('dev@example.com',)
app_1             | DEBUG 2022-06-30 17:53:24,919 django.db.backends (0.001) SELECT "experiments_nimbusexperiment"."id", "experiments_nimbusexperiment"."parent_id", "experiments_nimbusexperiment"."is_rollout", "experiments_nimbusexperiment"."is_archived", "experiments_nimbusexperiment"."owner_id", "experiments_nimbusexperiment"."status", "experiments_nimbusexperiment"."status_next", "experiments_nimbusexperiment"."publish_status", "experiments_nimbusexperiment"."name", "experiments_nimbusexperiment"."slug", "experiments_nimbusexperiment"."public_description", "experiments_nimbusexperiment"."risk_mitigation_link", "experiments_nimbusexperiment"."is_paused", "experiments_nimbusexperiment"."proposed_duration", "experiments_nimbusexperiment"."proposed_enrollment", "experiments_nimbusexperiment"."population_percent", "experiments_nimbusexperiment"."total_enrolled_clients", "experiments_nimbusexperiment"."firefox_min_version", "experiments_nimbusexperiment"."firefox_max_version", "experiments_nimbusexperiment"."application", "experiments_nimbusexperiment"."channel", "experiments_nimbusexperiment"."is_sticky", "experiments_nimbusexperiment"."hypothesis", "experiments_nimbusexperiment"."primary_outcomes", "experiments_nimbusexperiment"."secondary_outcomes", "experiments_nimbusexperiment"."warn_feature_schema", "experiments_nimbusexperiment"."targeting_config_slug", "experiments_nimbusexperiment"."reference_branch_id", "experiments_nimbusexperiment"."published_dto", "experiments_nimbusexperiment"."results_data", "experiments_nimbusexperiment"."risk_partner_related", "experiments_nimbusexperiment"."risk_revenue", "experiments_nimbusexperiment"."risk_brand", "experiments_nimbusexperiment"."conclusion_recommendation", "experiments_nimbusexperiment"."takeaways_summary" FROM "experiments_nimbusexperiment" WHERE "experiments_nimbusexperiment"."slug" = 'progressive-247-task-force' LIMIT 21; args=('progressive-247-task-force',)
app_1             | DEBUG 2022-06-30 17:53:24,921 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,924 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,926 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,928 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,930 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,932 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,934 django.db.backends (0.001) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."id" = 23 LIMIT 21; args=(23,)
app_1             | DEBUG 2022-06-30 17:53:24,935 django.db.backends (0.001) SELECT "experiments_nimbusbranch"."id", "experiments_nimbusbranch"."experiment_id", "experiments_nimbusbranch"."name", "experiments_nimbusbranch"."slug", "experiments_nimbusbranch"."description", "experiments_nimbusbranch"."ratio" FROM "experiments_nimbusbranch" WHERE "experiments_nimbusbranch"."id" = 43 LIMIT 21; args=(43,)
app_1             | DEBUG 2022-06-30 17:53:24,936 django.db.backends (0.000) SELECT (1) AS "a" FROM "experiments_nimbusbranch" WHERE "experiments_nimbusbranch"."experiment_id" = 22 LIMIT 1; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,937 django.db.backends (0.001) SELECT "experiments_nimbusbranch"."id", "experiments_nimbusbranch"."experiment_id", "experiments_nimbusbranch"."name", "experiments_nimbusbranch"."slug", "experiments_nimbusbranch"."description", "experiments_nimbusbranch"."ratio" FROM "experiments_nimbusbranch" WHERE ("experiments_nimbusbranch"."experiment_id" = 22 AND NOT ("experiments_nimbusbranch"."id" = 43)) ORDER BY "experiments_nimbusbranch"."id" ASC; args=(22, 43)
app_1             | DEBUG 2022-06-30 17:53:24,941 django.db.backends (0.001) SELECT "experiments_nimbusdocumentationlink"."id", "experiments_nimbusdocumentationlink"."experiment_id", "experiments_nimbusdocumentationlink"."title", "experiments_nimbusdocumentationlink"."link" FROM "experiments_nimbusdocumentationlink" WHERE "experiments_nimbusdocumentationlink"."experiment_id" = 22 ORDER BY "experiments_nimbusdocumentationlink"."id" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,945 django.db.backends (0.003) SELECT "experiments_nimbusbranchscreenshot"."id", "experiments_nimbusbranchscreenshot"."branch_id", "experiments_nimbusbranchscreenshot"."image", "experiments_nimbusbranchscreenshot"."description" FROM "experiments_nimbusbranchscreenshot" WHERE "experiments_nimbusbranchscreenshot"."branch_id" = 43 ORDER BY "experiments_nimbusbranchscreenshot"."id" ASC; args=(43,)
app_1             | DEBUG 2022-06-30 17:53:24,947 django.db.backends (0.002) SELECT "experiments_nimbusbranchfeaturevalue"."id", "experiments_nimbusbranchfeaturevalue"."branch_id", "experiments_nimbusbranchfeaturevalue"."feature_config_id", "experiments_nimbusbranchfeaturevalue"."enabled", "experiments_nimbusbranchfeaturevalue"."value" FROM "experiments_nimbusbranchfeaturevalue" WHERE "experiments_nimbusbranchfeaturevalue"."branch_id" = 43; args=(43,)
app_1             | DEBUG 2022-06-30 17:53:24,950 django.db.backends (0.001) SELECT (1) AS "a" FROM "experiments_nimbusbranchfeaturevalue" WHERE "experiments_nimbusbranchfeaturevalue"."branch_id" = 43 LIMIT 1; args=(43,)
app_1             | DEBUG 2022-06-30 17:53:24,954 django.db.backends (0.001) SELECT "experiments_nimbusbranchfeaturevalue"."id", "experiments_nimbusbranchfeaturevalue"."branch_id", "experiments_nimbusbranchfeaturevalue"."feature_config_id", "experiments_nimbusbranchfeaturevalue"."enabled", "experiments_nimbusbranchfeaturevalue"."value" FROM "experiments_nimbusbranchfeaturevalue" LEFT OUTER JOIN "experiments_nimbusfeatureconfig" ON ("experiments_nimbusbranchfeaturevalue"."feature_config_id" = "experiments_nimbusfeatureconfig"."id") WHERE "experiments_nimbusbranchfeaturevalue"."branch_id" = 43 ORDER BY "experiments_nimbusfeatureconfig"."slug" ASC LIMIT 1; args=(43,)
app_1             | DEBUG 2022-06-30 17:53:24,956 django.db.backends (0.001) SELECT "experiments_nimbusbranch"."id", "experiments_nimbusbranch"."experiment_id", "experiments_nimbusbranch"."name", "experiments_nimbusbranch"."slug", "experiments_nimbusbranch"."description", "experiments_nimbusbranch"."ratio" FROM "experiments_nimbusbranch" WHERE ("experiments_nimbusbranch"."experiment_id" = 22 AND NOT ("experiments_nimbusbranch"."id" = 43)) ORDER BY "experiments_nimbusbranch"."id" ASC; args=(22, 43)
app_1             | DEBUG 2022-06-30 17:53:24,958 django.db.backends (0.001) SELECT "experiments_nimbusbranchscreenshot"."id", "experiments_nimbusbranchscreenshot"."branch_id", "experiments_nimbusbranchscreenshot"."image", "experiments_nimbusbranchscreenshot"."description" FROM "experiments_nimbusbranchscreenshot" WHERE "experiments_nimbusbranchscreenshot"."branch_id" = 44 ORDER BY "experiments_nimbusbranchscreenshot"."id" ASC; args=(44,)
app_1             | DEBUG 2022-06-30 17:53:24,959 django.db.backends (0.000) SELECT "experiments_nimbusbranchfeaturevalue"."id", "experiments_nimbusbranchfeaturevalue"."branch_id", "experiments_nimbusbranchfeaturevalue"."feature_config_id", "experiments_nimbusbranchfeaturevalue"."enabled", "experiments_nimbusbranchfeaturevalue"."value" FROM "experiments_nimbusbranchfeaturevalue" WHERE "experiments_nimbusbranchfeaturevalue"."branch_id" = 44; args=(44,)
app_1             | DEBUG 2022-06-30 17:53:24,960 django.db.backends (0.001) SELECT (1) AS "a" FROM "experiments_nimbusbranchfeaturevalue" WHERE "experiments_nimbusbranchfeaturevalue"."branch_id" = 44 LIMIT 1; args=(44,)
app_1             | DEBUG 2022-06-30 17:53:24,962 django.db.backends (0.001) SELECT "experiments_nimbusbranchfeaturevalue"."id", "experiments_nimbusbranchfeaturevalue"."branch_id", "experiments_nimbusbranchfeaturevalue"."feature_config_id", "experiments_nimbusbranchfeaturevalue"."enabled", "experiments_nimbusbranchfeaturevalue"."value" FROM "experiments_nimbusbranchfeaturevalue" LEFT OUTER JOIN "experiments_nimbusfeatureconfig" ON ("experiments_nimbusbranchfeaturevalue"."feature_config_id" = "experiments_nimbusfeatureconfig"."id") WHERE "experiments_nimbusbranchfeaturevalue"."branch_id" = 44 ORDER BY "experiments_nimbusfeatureconfig"."slug" ASC LIMIT 1; args=(44,)
app_1             | DEBUG 2022-06-30 17:53:24,964 django.db.backends (0.001) SELECT "experiments_nimbusfeatureconfig"."id", "experiments_nimbusfeatureconfig"."name", "experiments_nimbusfeatureconfig"."slug", "experiments_nimbusfeatureconfig"."description", "experiments_nimbusfeatureconfig"."application", "experiments_nimbusfeatureconfig"."owner_email", "experiments_nimbusfeatureconfig"."schema", "experiments_nimbusfeatureconfig"."read_only" FROM "experiments_nimbusfeatureconfig" INNER JOIN "experiments_nimbusexperiment_feature_configs" ON ("experiments_nimbusfeatureconfig"."id" = "experiments_nimbusexperiment_feature_configs"."nimbusfeatureconfig_id") WHERE "experiments_nimbusexperiment_feature_configs"."nimbusexperiment_id" = 22; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,969 django.db.backends (0.004) SELECT "base_locale"."id", "base_locale"."code", "base_locale"."name" FROM "base_locale" INNER JOIN "experiments_nimbusexperiment_locales" ON ("base_locale"."id" = "experiments_nimbusexperiment_locales"."locale_id") WHERE "experiments_nimbusexperiment_locales"."nimbusexperiment_id" = 22 ORDER BY "base_locale"."name" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,970 django.db.backends (0.001) SELECT "base_country"."id", "base_country"."code", "base_country"."name" FROM "base_country" INNER JOIN "experiments_nimbusexperiment_countries" ON ("base_country"."id" = "experiments_nimbusexperiment_countries"."country_id") WHERE "experiments_nimbusexperiment_countries"."nimbusexperiment_id" = 22 ORDER BY "base_country"."name" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,971 django.db.backends (0.001) SELECT "base_language"."id", "base_language"."code", "base_language"."name" FROM "base_language" INNER JOIN "experiments_nimbusexperiment_languages" ON ("base_language"."id" = "experiments_nimbusexperiment_languages"."language_id") WHERE "experiments_nimbusexperiment_languages"."nimbusexperiment_id" = 22 ORDER BY "base_language"."name" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,973 django.db.backends (0.001) SELECT "projects_project"."id", "projects_project"."name", "projects_project"."slug" FROM "projects_project" INNER JOIN "experiments_nimbusexperiment_projects" ON ("projects_project"."id" = "experiments_nimbusexperiment_projects"."project_id") WHERE "experiments_nimbusexperiment_projects"."nimbusexperiment_id" = 22 ORDER BY "projects_project"."name" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,974 django.db.backends (0.000) SELECT (1) AS "a" FROM "experiments_nimbusfeatureconfig" INNER JOIN "experiments_nimbusexperiment_feature_configs" ON ("experiments_nimbusfeatureconfig"."id" = "experiments_nimbusexperiment_feature_configs"."nimbusfeatureconfig_id") WHERE "experiments_nimbusexperiment_feature_configs"."nimbusexperiment_id" = 22 LIMIT 1; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,976 django.db.backends (0.002) SELECT "experiments_nimbusfeatureconfig"."id", "experiments_nimbusfeatureconfig"."name", "experiments_nimbusfeatureconfig"."slug", "experiments_nimbusfeatureconfig"."description", "experiments_nimbusfeatureconfig"."application", "experiments_nimbusfeatureconfig"."owner_email", "experiments_nimbusfeatureconfig"."schema", "experiments_nimbusfeatureconfig"."read_only" FROM "experiments_nimbusfeatureconfig" INNER JOIN "experiments_nimbusexperiment_feature_configs" ON ("experiments_nimbusfeatureconfig"."id" = "experiments_nimbusexperiment_feature_configs"."nimbusfeatureconfig_id") WHERE "experiments_nimbusexperiment_feature_configs"."nimbusexperiment_id" = 22 ORDER BY "experiments_nimbusfeatureconfig"."slug" ASC LIMIT 1; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,980 django.db.backends (0.000) SELECT "experiments_nimbusbranch"."id", "experiments_nimbusbranch"."experiment_id", "experiments_nimbusbranch"."name", "experiments_nimbusbranch"."slug", "experiments_nimbusbranch"."description", "experiments_nimbusbranch"."ratio" FROM "experiments_nimbusbranch" WHERE "experiments_nimbusbranch"."id" = 43 LIMIT 21; args=(43,)
app_1             | DEBUG 2022-06-30 17:53:24,982 django.db.backends (0.002) SELECT "experiments_nimbusfeatureconfig"."id", "experiments_nimbusfeatureconfig"."name", "experiments_nimbusfeatureconfig"."slug", "experiments_nimbusfeatureconfig"."description", "experiments_nimbusfeatureconfig"."application", "experiments_nimbusfeatureconfig"."owner_email", "experiments_nimbusfeatureconfig"."schema", "experiments_nimbusfeatureconfig"."read_only" FROM "experiments_nimbusfeatureconfig" WHERE "experiments_nimbusfeatureconfig"."id" = 45 LIMIT 21; args=(45,)
app_1             | DEBUG 2022-06-30 17:53:24,984 django.db.backends (0.001) SELECT "experiments_nimbusbranchfeaturevalue"."id", "experiments_nimbusbranchfeaturevalue"."branch_id", "experiments_nimbusbranchfeaturevalue"."feature_config_id", "experiments_nimbusbranchfeaturevalue"."enabled", "experiments_nimbusbranchfeaturevalue"."value" FROM "experiments_nimbusbranchfeaturevalue" WHERE "experiments_nimbusbranchfeaturevalue"."id" = 43 LIMIT 21; args=(43,)
app_1             | DEBUG 2022-06-30 17:53:24,985 django.db.backends (0.001) SELECT "experiments_nimbusbranch"."id", "experiments_nimbusbranch"."experiment_id", "experiments_nimbusbranch"."name", "experiments_nimbusbranch"."slug", "experiments_nimbusbranch"."description", "experiments_nimbusbranch"."ratio" FROM "experiments_nimbusbranch" WHERE "experiments_nimbusbranch"."id" = 43 LIMIT 21; args=(43,)
app_1             | DEBUG 2022-06-30 17:53:24,986 django.db.backends (0.001) SELECT "experiments_nimbusexperiment"."id", "experiments_nimbusexperiment"."parent_id", "experiments_nimbusexperiment"."is_rollout", "experiments_nimbusexperiment"."is_archived", "experiments_nimbusexperiment"."owner_id", "experiments_nimbusexperiment"."status", "experiments_nimbusexperiment"."status_next", "experiments_nimbusexperiment"."publish_status", "experiments_nimbusexperiment"."name", "experiments_nimbusexperiment"."slug", "experiments_nimbusexperiment"."public_description", "experiments_nimbusexperiment"."risk_mitigation_link", "experiments_nimbusexperiment"."is_paused", "experiments_nimbusexperiment"."proposed_duration", "experiments_nimbusexperiment"."proposed_enrollment", "experiments_nimbusexperiment"."population_percent", "experiments_nimbusexperiment"."total_enrolled_clients", "experiments_nimbusexperiment"."firefox_min_version", "experiments_nimbusexperiment"."firefox_max_version", "experiments_nimbusexperiment"."application", "experiments_nimbusexperiment"."channel", "experiments_nimbusexperiment"."is_sticky", "experiments_nimbusexperiment"."hypothesis", "experiments_nimbusexperiment"."primary_outcomes", "experiments_nimbusexperiment"."secondary_outcomes", "experiments_nimbusexperiment"."warn_feature_schema", "experiments_nimbusexperiment"."targeting_config_slug", "experiments_nimbusexperiment"."reference_branch_id", "experiments_nimbusexperiment"."published_dto", "experiments_nimbusexperiment"."results_data", "experiments_nimbusexperiment"."risk_partner_related", "experiments_nimbusexperiment"."risk_revenue", "experiments_nimbusexperiment"."risk_brand", "experiments_nimbusexperiment"."conclusion_recommendation", "experiments_nimbusexperiment"."takeaways_summary" FROM "experiments_nimbusexperiment" WHERE "experiments_nimbusexperiment"."id" = 22 LIMIT 21; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,988 django.db.backends (0.001) SELECT COUNT(*) AS "__count" FROM "experiments_nimbusfeatureconfig" INNER JOIN "experiments_nimbusexperiment_feature_configs" ON ("experiments_nimbusfeatureconfig"."id" = "experiments_nimbusexperiment_feature_configs"."nimbusfeatureconfig_id") WHERE "experiments_nimbusexperiment_feature_configs"."nimbusexperiment_id" = 22; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,989 django.db.backends (0.001) SELECT "experiments_nimbusbranch"."id", "experiments_nimbusbranch"."experiment_id", "experiments_nimbusbranch"."name", "experiments_nimbusbranch"."slug", "experiments_nimbusbranch"."description", "experiments_nimbusbranch"."ratio" FROM "experiments_nimbusbranch" WHERE "experiments_nimbusbranch"."id" = 44 LIMIT 21; args=(44,)
app_1             | DEBUG 2022-06-30 17:53:24,990 django.db.backends (0.000) SELECT "experiments_nimbusfeatureconfig"."id", "experiments_nimbusfeatureconfig"."name", "experiments_nimbusfeatureconfig"."slug", "experiments_nimbusfeatureconfig"."description", "experiments_nimbusfeatureconfig"."application", "experiments_nimbusfeatureconfig"."owner_email", "experiments_nimbusfeatureconfig"."schema", "experiments_nimbusfeatureconfig"."read_only" FROM "experiments_nimbusfeatureconfig" WHERE "experiments_nimbusfeatureconfig"."id" = 45 LIMIT 21; args=(45,)
app_1             | DEBUG 2022-06-30 17:53:24,991 django.db.backends (0.000) SELECT "experiments_nimbusbranchfeaturevalue"."id", "experiments_nimbusbranchfeaturevalue"."branch_id", "experiments_nimbusbranchfeaturevalue"."feature_config_id", "experiments_nimbusbranchfeaturevalue"."enabled", "experiments_nimbusbranchfeaturevalue"."value" FROM "experiments_nimbusbranchfeaturevalue" WHERE "experiments_nimbusbranchfeaturevalue"."id" = 44 LIMIT 21; args=(44,)
app_1             | DEBUG 2022-06-30 17:53:24,992 django.db.backends (0.000) SELECT "experiments_nimbusbranch"."id", "experiments_nimbusbranch"."experiment_id", "experiments_nimbusbranch"."name", "experiments_nimbusbranch"."slug", "experiments_nimbusbranch"."description", "experiments_nimbusbranch"."ratio" FROM "experiments_nimbusbranch" WHERE "experiments_nimbusbranch"."id" = 44 LIMIT 21; args=(44,)
app_1             | DEBUG 2022-06-30 17:53:24,993 django.db.backends (0.000) SELECT "experiments_nimbusexperiment"."id", "experiments_nimbusexperiment"."parent_id", "experiments_nimbusexperiment"."is_rollout", "experiments_nimbusexperiment"."is_archived", "experiments_nimbusexperiment"."owner_id", "experiments_nimbusexperiment"."status", "experiments_nimbusexperiment"."status_next", "experiments_nimbusexperiment"."publish_status", "experiments_nimbusexperiment"."name", "experiments_nimbusexperiment"."slug", "experiments_nimbusexperiment"."public_description", "experiments_nimbusexperiment"."risk_mitigation_link", "experiments_nimbusexperiment"."is_paused", "experiments_nimbusexperiment"."proposed_duration", "experiments_nimbusexperiment"."proposed_enrollment", "experiments_nimbusexperiment"."population_percent", "experiments_nimbusexperiment"."total_enrolled_clients", "experiments_nimbusexperiment"."firefox_min_version", "experiments_nimbusexperiment"."firefox_max_version", "experiments_nimbusexperiment"."application", "experiments_nimbusexperiment"."channel", "experiments_nimbusexperiment"."is_sticky", "experiments_nimbusexperiment"."hypothesis", "experiments_nimbusexperiment"."primary_outcomes", "experiments_nimbusexperiment"."secondary_outcomes", "experiments_nimbusexperiment"."warn_feature_schema", "experiments_nimbusexperiment"."targeting_config_slug", "experiments_nimbusexperiment"."reference_branch_id", "experiments_nimbusexperiment"."published_dto", "experiments_nimbusexperiment"."results_data", "experiments_nimbusexperiment"."risk_partner_related", "experiments_nimbusexperiment"."risk_revenue", "experiments_nimbusexperiment"."risk_brand", "experiments_nimbusexperiment"."conclusion_recommendation", "experiments_nimbusexperiment"."takeaways_summary" FROM "experiments_nimbusexperiment" WHERE "experiments_nimbusexperiment"."id" = 22 LIMIT 21; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,994 django.db.backends (0.000) SELECT COUNT(*) AS "__count" FROM "experiments_nimbusfeatureconfig" INNER JOIN "experiments_nimbusexperiment_feature_configs" ON ("experiments_nimbusfeatureconfig"."id" = "experiments_nimbusexperiment_feature_configs"."nimbusfeatureconfig_id") WHERE "experiments_nimbusexperiment_feature_configs"."nimbusexperiment_id" = 22; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:24,995 django.db.backends (0.000) SELECT "experiments_nimbusfeatureconfig"."id", "experiments_nimbusfeatureconfig"."name", "experiments_nimbusfeatureconfig"."slug", "experiments_nimbusfeatureconfig"."description", "experiments_nimbusfeatureconfig"."application", "experiments_nimbusfeatureconfig"."owner_email", "experiments_nimbusfeatureconfig"."schema", "experiments_nimbusfeatureconfig"."read_only" FROM "experiments_nimbusfeatureconfig" WHERE "experiments_nimbusfeatureconfig"."id" = 45 LIMIT 21; args=(45,)
app_1             | DEBUG 2022-06-30 17:53:24,996 django.db.backends (0.000) SELECT "experiments_nimbusfeatureconfig"."id", "experiments_nimbusfeatureconfig"."name", "experiments_nimbusfeatureconfig"."slug", "experiments_nimbusfeatureconfig"."description", "experiments_nimbusfeatureconfig"."application", "experiments_nimbusfeatureconfig"."owner_email", "experiments_nimbusfeatureconfig"."schema", "experiments_nimbusfeatureconfig"."read_only" FROM "experiments_nimbusfeatureconfig" WHERE "experiments_nimbusfeatureconfig"."id" = 45 LIMIT 21; args=(45,)
app_1             | DEBUG 2022-06-30 17:53:24,997 django.db.backends (0.001) SELECT (1) AS "a" FROM "experiments_nimbusexperiment" WHERE ("experiments_nimbusexperiment"."name" = 'Progressive 24/7 task-force' AND NOT ("experiments_nimbusexperiment"."id" = 22)) LIMIT 1; args=('Progressive 24/7 task-force', 22)
app_1             | DEBUG 2022-06-30 17:53:24,999 django.db.backends (0.001) SELECT (1) AS "a" FROM "experiments_nimbusexperiment" WHERE ("experiments_nimbusexperiment"."slug" = 'progressive-247-task-force' AND NOT ("experiments_nimbusexperiment"."id" = 22)) LIMIT 1; args=('progressive-247-task-force', 22)
app_1             | DEBUG 2022-06-30 17:53:25,000 django.db.backends (0.001) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."id" = 23 LIMIT 21; args=(23,)
app_1             | DEBUG 2022-06-30 17:53:25,001 django.db.backends (0.001) SELECT "base_locale"."id", "base_locale"."code", "base_locale"."name" FROM "base_locale" WHERE "base_locale"."id" = 1 LIMIT 21; args=(1,)
app_1             | DEBUG 2022-06-30 17:53:25,003 django.db.backends (0.001) SELECT "base_locale"."id", "base_locale"."code", "base_locale"."name" FROM "base_locale" WHERE "base_locale"."id" = 2 LIMIT 21; args=(2,)
app_1             | DEBUG 2022-06-30 17:53:25,004 django.db.backends (0.001) SELECT "base_locale"."id", "base_locale"."code", "base_locale"."name" FROM "base_locale" WHERE "base_locale"."id" = 129 LIMIT 21; args=(129,)
app_1             | DEBUG 2022-06-30 17:53:25,006 django.db.backends (0.001) SELECT "base_country"."id", "base_country"."code", "base_country"."name" FROM "base_country" WHERE "base_country"."id" = 1 LIMIT 21; args=(1,)
app_1             | DEBUG 2022-06-30 17:53:25,007 django.db.backends (0.001) SELECT "base_country"."id", "base_country"."code", "base_country"."name" FROM "base_country" WHERE "base_country"."id" = 2 LIMIT 21; args=(2,)
app_1             | DEBUG 2022-06-30 17:53:25,008 django.db.backends (0.000) SELECT "base_country"."id", "base_country"."code", "base_country"."name" FROM "base_country" WHERE "base_country"."id" = 3 LIMIT 21; args=(3,)
app_1             | DEBUG 2022-06-30 17:53:25,009 django.db.backends (0.000) SELECT "base_language"."id", "base_language"."code", "base_language"."name" FROM "base_language" WHERE "base_language"."id" = 2 LIMIT 21; args=(2,)
app_1             | DEBUG 2022-06-30 17:53:25,010 django.db.backends (0.000) SELECT "base_language"."id", "base_language"."code", "base_language"."name" FROM "base_language" WHERE "base_language"."id" = 1 LIMIT 21; args=(1,)
app_1             | DEBUG 2022-06-30 17:53:25,011 django.db.backends (0.001) SELECT "base_language"."id", "base_language"."code", "base_language"."name" FROM "base_language" WHERE "base_language"."id" = 4 LIMIT 21; args=(4,)
app_1             | DEBUG 2022-06-30 17:53:25,013 django.db.backends (0.001) SELECT "projects_project"."id", "projects_project"."name", "projects_project"."slug" FROM "projects_project" WHERE "projects_project"."id" = 64 LIMIT 21; args=(64,)
app_1             | DEBUG 2022-06-30 17:53:25,015 django.db.backends (0.001) SELECT "projects_project"."id", "projects_project"."name", "projects_project"."slug" FROM "projects_project" WHERE "projects_project"."id" = 65 LIMIT 21; args=(65,)
app_1             | DEBUG 2022-06-30 17:53:25,016 django.db.backends (0.001) SELECT "projects_project"."id", "projects_project"."name", "projects_project"."slug" FROM "projects_project" WHERE "projects_project"."id" = 66 LIMIT 21; args=(66,)
app_1             | DEBUG 2022-06-30 17:53:25,019 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,021 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,023 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,025 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,027 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE ("experiments_nimbuschangelog"."experiment_id" = 22 AND ("experiments_nimbuschangelog"."experiment_data" -> 'is_paused') = 'true' AND "experiments_nimbuschangelog"."new_publish_status" = 'Idle' AND "experiments_nimbuschangelog"."new_status" = 'Live' AND "experiments_nimbuschangelog"."new_status_next" IS NULL) ORDER BY "experiments_nimbuschangelog"."changed_on" ASC LIMIT 1; args=(22, 'is_paused', 'true', <PublishStatus.IDLE: 'Idle'>, <Status.LIVE: 'Live'>)
app_1             | DEBUG 2022-06-30 17:53:25,029 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,030 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,033 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,035 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,037 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,039 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,041 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,043 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,044 django.db.backends (0.000) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE ("experiments_nimbuschangelog"."experiment_id" = 22 AND "experiments_nimbuschangelog"."new_publish_status" = 'Review' AND "experiments_nimbuschangelog"."old_publish_status" = 'Idle') ORDER BY "experiments_nimbuschangelog"."changed_on" DESC LIMIT 1; args=(22, <PublishStatus.REVIEW: 'Review'>, <PublishStatus.IDLE: 'Idle'>)
app_1             | DEBUG 2022-06-30 17:53:25,046 django.db.backends (0.000) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" DESC LIMIT 1; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,047 django.db.backends (0.000) SELECT (1) AS "a" FROM "experiments_nimbuschangelog" WHERE ("experiments_nimbuschangelog"."id" = 149 AND "experiments_nimbuschangelog"."old_status" = "experiments_nimbuschangelog"."new_status" AND "experiments_nimbuschangelog"."new_publish_status" = 'Idle' AND "experiments_nimbuschangelog"."old_publish_status" IN ('Review', 'Waiting') AND NOT "experiments_nimbuschangelog"."published_dto_changed") LIMIT 1; args=(149, <PublishStatus.IDLE: 'Idle'>, <PublishStatus.REVIEW: 'Review'>, <PublishStatus.WAITING: 'Waiting'>)
app_1             | DEBUG 2022-06-30 17:53:25,049 django.db.backends (0.001) SELECT "experiments_nimbuschangelog"."id", "experiments_nimbuschangelog"."experiment_id", "experiments_nimbuschangelog"."changed_on", "experiments_nimbuschangelog"."changed_by_id", "experiments_nimbuschangelog"."old_status", "experiments_nimbuschangelog"."old_status_next", "experiments_nimbuschangelog"."old_publish_status", "experiments_nimbuschangelog"."new_status", "experiments_nimbuschangelog"."new_status_next", "experiments_nimbuschangelog"."new_publish_status", "experiments_nimbuschangelog"."message", "experiments_nimbuschangelog"."experiment_data", "experiments_nimbuschangelog"."published_dto_changed" FROM "experiments_nimbuschangelog" WHERE "experiments_nimbuschangelog"."experiment_id" = 22 ORDER BY "experiments_nimbuschangelog"."changed_on" DESC LIMIT 1; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,051 django.db.backends (0.001) SELECT (1) AS "a" FROM "experiments_nimbuschangelog" WHERE ("experiments_nimbuschangelog"."id" = 149 AND "experiments_nimbuschangelog"."old_status" = "experiments_nimbuschangelog"."new_status" AND "experiments_nimbuschangelog"."new_publish_status" = 'Review' AND "experiments_nimbuschangelog"."old_publish_status" = 'Waiting') LIMIT 1; args=(149, <PublishStatus.REVIEW: 'Review'>, <PublishStatus.WAITING: 'Waiting'>)
app_1             | DEBUG 2022-06-30 17:53:25,053 django.db.backends (0.000) SELECT (1) AS "a" FROM "experiments_nimbusbranchfeaturevalue" WHERE "experiments_nimbusbranchfeaturevalue"."branch_id" = 43 LIMIT 1; args=(43,)
app_1             | DEBUG 2022-06-30 17:53:25,054 django.db.backends (0.001) SELECT "experiments_nimbusbranchfeaturevalue"."id", "experiments_nimbusbranchfeaturevalue"."branch_id", "experiments_nimbusbranchfeaturevalue"."feature_config_id", "experiments_nimbusbranchfeaturevalue"."enabled", "experiments_nimbusbranchfeaturevalue"."value" FROM "experiments_nimbusbranchfeaturevalue" LEFT OUTER JOIN "experiments_nimbusfeatureconfig" ON ("experiments_nimbusbranchfeaturevalue"."feature_config_id" = "experiments_nimbusfeatureconfig"."id") WHERE "experiments_nimbusbranchfeaturevalue"."branch_id" = 43 ORDER BY "experiments_nimbusfeatureconfig"."slug" ASC LIMIT 1; args=(43,)
app_1             | DEBUG 2022-06-30 17:53:25,055 django.db.backends (0.001) SELECT (1) AS "a" FROM "experiments_nimbusbranchfeaturevalue" WHERE "experiments_nimbusbranchfeaturevalue"."branch_id" = 43 LIMIT 1; args=(43,)
app_1             | DEBUG 2022-06-30 17:53:25,057 django.db.backends (0.001) SELECT "experiments_nimbusbranchfeaturevalue"."id", "experiments_nimbusbranchfeaturevalue"."branch_id", "experiments_nimbusbranchfeaturevalue"."feature_config_id", "experiments_nimbusbranchfeaturevalue"."enabled", "experiments_nimbusbranchfeaturevalue"."value" FROM "experiments_nimbusbranchfeaturevalue" LEFT OUTER JOIN "experiments_nimbusfeatureconfig" ON ("experiments_nimbusbranchfeaturevalue"."feature_config_id" = "experiments_nimbusfeatureconfig"."id") WHERE "experiments_nimbusbranchfeaturevalue"."branch_id" = 43 ORDER BY "experiments_nimbusfeatureconfig"."slug" ASC LIMIT 1; args=(43,)
app_1             | DEBUG 2022-06-30 17:53:25,059 django.db.backends (0.000) SELECT (1) AS "a" FROM "experiments_nimbusbranchfeaturevalue" WHERE "experiments_nimbusbranchfeaturevalue"."branch_id" = 44 LIMIT 1; args=(44,)
app_1             | DEBUG 2022-06-30 17:53:25,061 django.db.backends (0.001) SELECT "experiments_nimbusbranchfeaturevalue"."id", "experiments_nimbusbranchfeaturevalue"."branch_id", "experiments_nimbusbranchfeaturevalue"."feature_config_id", "experiments_nimbusbranchfeaturevalue"."enabled", "experiments_nimbusbranchfeaturevalue"."value" FROM "experiments_nimbusbranchfeaturevalue" LEFT OUTER JOIN "experiments_nimbusfeatureconfig" ON ("experiments_nimbusbranchfeaturevalue"."feature_config_id" = "experiments_nimbusfeatureconfig"."id") WHERE "experiments_nimbusbranchfeaturevalue"."branch_id" = 44 ORDER BY "experiments_nimbusfeatureconfig"."slug" ASC LIMIT 1; args=(44,)
app_1             | DEBUG 2022-06-30 17:53:25,062 django.db.backends (0.001) SELECT (1) AS "a" FROM "experiments_nimbusbranchfeaturevalue" WHERE "experiments_nimbusbranchfeaturevalue"."branch_id" = 44 LIMIT 1; args=(44,)
app_1             | DEBUG 2022-06-30 17:53:25,064 django.db.backends (0.001) SELECT "experiments_nimbusbranchfeaturevalue"."id", "experiments_nimbusbranchfeaturevalue"."branch_id", "experiments_nimbusbranchfeaturevalue"."feature_config_id", "experiments_nimbusbranchfeaturevalue"."enabled", "experiments_nimbusbranchfeaturevalue"."value" FROM "experiments_nimbusbranchfeaturevalue" LEFT OUTER JOIN "experiments_nimbusfeatureconfig" ON ("experiments_nimbusbranchfeaturevalue"."feature_config_id" = "experiments_nimbusfeatureconfig"."id") WHERE "experiments_nimbusbranchfeaturevalue"."branch_id" = 44 ORDER BY "experiments_nimbusfeatureconfig"."slug" ASC LIMIT 1; args=(44,)
app_1             | DEBUG 2022-06-30 17:53:25,066 django.db.backends (0.001) SELECT "experiments_nimbusfeatureconfig"."id", "experiments_nimbusfeatureconfig"."name", "experiments_nimbusfeatureconfig"."slug", "experiments_nimbusfeatureconfig"."description", "experiments_nimbusfeatureconfig"."application", "experiments_nimbusfeatureconfig"."owner_email", "experiments_nimbusfeatureconfig"."schema", "experiments_nimbusfeatureconfig"."read_only" FROM "experiments_nimbusfeatureconfig" INNER JOIN "experiments_nimbusexperiment_feature_configs" ON ("experiments_nimbusfeatureconfig"."id" = "experiments_nimbusexperiment_feature_configs"."nimbusfeatureconfig_id") WHERE "experiments_nimbusexperiment_feature_configs"."nimbusexperiment_id" = 22; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,068 django.db.backends (0.000) SELECT "experiments_nimbusdocumentationlink"."id", "experiments_nimbusdocumentationlink"."experiment_id", "experiments_nimbusdocumentationlink"."title", "experiments_nimbusdocumentationlink"."link" FROM "experiments_nimbusdocumentationlink" WHERE "experiments_nimbusdocumentationlink"."experiment_id" = 22 ORDER BY "experiments_nimbusdocumentationlink"."id" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,070 django.db.backends (0.001) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."id" = 23 LIMIT 21; args=(23,)
app_1             | DEBUG 2022-06-30 17:53:25,071 django.db.backends (0.000) SELECT "base_locale"."id", "base_locale"."code", "base_locale"."name" FROM "base_locale" INNER JOIN "experiments_nimbusexperiment_locales" ON ("base_locale"."id" = "experiments_nimbusexperiment_locales"."locale_id") WHERE "experiments_nimbusexperiment_locales"."nimbusexperiment_id" = 22 ORDER BY "base_locale"."name" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,073 django.db.backends (0.001) SELECT "base_country"."id", "base_country"."code", "base_country"."name" FROM "base_country" INNER JOIN "experiments_nimbusexperiment_countries" ON ("base_country"."id" = "experiments_nimbusexperiment_countries"."country_id") WHERE "experiments_nimbusexperiment_countries"."nimbusexperiment_id" = 22 ORDER BY "base_country"."name" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,074 django.db.backends (0.000) SELECT "base_language"."id", "base_language"."code", "base_language"."name" FROM "base_language" INNER JOIN "experiments_nimbusexperiment_languages" ON ("base_language"."id" = "experiments_nimbusexperiment_languages"."language_id") WHERE "experiments_nimbusexperiment_languages"."nimbusexperiment_id" = 22 ORDER BY "base_language"."name" ASC; args=(22,)
app_1             | DEBUG 2022-06-30 17:53:25,076 django.db.backends (0.001) SELECT "experiments_nimbusbranchscreenshot"."id", "experiments_nimbusbranchscreenshot"."branch_id", "experiments_nimbusbranchscreenshot"."image", "experiments_nimbusbranchscreenshot"."description" FROM "experiments_nimbusbranchscreenshot" WHERE "experiments_nimbusbranchscreenshot"."branch_id" = 43 ORDER BY "experiments_nimbusbranchscreenshot"."id" ASC; args=(43,)
app_1             | DEBUG 2022-06-30 17:53:25,077 django.db.backends (0.001) SELECT "experiments_nimbusbranchscreenshot"."id", "experiments_nimbusbranchscreenshot"."branch_id", "experiments_nimbusbranchscreenshot"."image", "experiments_nimbusbranchscreenshot"."description" FROM "experiments_nimbusbranchscreenshot" WHERE "experiments_nimbusbranchscreenshot"."branch_id" = 44 ORDER BY "experiments_nimbusbranchscreenshot"."id" ASC; args=(44,)
app_1             | INFO 2022-06-30 17:53:25,079 request.summary 
nginx_1           | 172.18.0.1 - - [30/Jun/2022:17:53:25 +0000] "POST /api/v5/graphql HTTP/1.1" 200 9820 "https://localhost/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:103.0) Gecko/20100101 Firefox/103.0" "-"
app_1             | DEBUG 2022-06-30 17:53:25,161 django.db.backends (0.001) SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" WHERE "auth_user"."username" = 'dev@example.com' LIMIT 21; args=('dev@example.com',)
app_1             | DEBUG 2022-06-30 17:53:25,164 django.db.backends (0.002) SELECT "experiments_nimbusexperiment"."id", "experiments_nimbusexperiment"."parent_id", "experiments_nimbusexperiment"."is_rollout", "experiments_nimbusexperiment"."is_archived", "experiments_nimbusexperiment"."owner_id", "experiments_nimbusexperiment"."status", "experiments_nimbusexperiment"."status_next", "experiments_nimbusexperiment"."publish_status", "experiments_nimbusexperiment"."name", "experiments_nimbusexperiment"."slug", "experiments_nimbusexperiment"."public_description", "experiments_nimbusexperiment"."risk_mitigation_link", "experiments_nimbusexperiment"."is_paused", "experiments_nimbusexperiment"."proposed_duration", "experiments_nimbusexperiment"."proposed_enrollment", "experiments_nimbusexperiment"."population_percent", "experiments_nimbusexperiment"."total_enrolled_clients", "experiments_nimbusexperiment"."firefox_min_version", "experiments_nimbusexperiment"."firefox_max_version", "experiments_nimbusexperiment"."application", "experiments_nimbusexperiment"."channel", "experiments_nimbusexperiment"."is_sticky", "experiments_nimbusexperiment"."hypothesis", "experiments_nimbusexperiment"."primary_outcomes", "experiments_nimbusexperiment"."secondary_outcomes", "experiments_nimbusexperiment"."warn_feature_schema", "experiments_nimbusexperiment"."targeting_config_slug", "experiments_nimbusexperiment"."reference_branch_id", "experiments_nimbusexperiment"."published_dto", "experiments_nimbusexperiment"."results_data", "experiments_nimbusexperiment"."risk_partner_related", "experiments_nimbusexperiment"."risk_revenue", "experiments_nimbusexperiment"."risk_brand", "experiments_nimbusexperiment"."conclusion_recommendation", "experiments_nimbusexperiment"."takeaways_summary" FROM "experiments_nimbusexperiment" WHERE "experiments_nimbusexperiment"."slug" = 'progressive-247-task-force' LIMIT 21; args=('progressive-247-task-force',)
```

Reply:
done, filing a separate ticket to tackle this, thank you @jaredlockhart 🙏